CHANGELOG.md

```diff
@@ -2,18 +2,11 @@
 
 ## v0.62.0
 
-This feature release of React Native includes many changes for the platform: it has been a lot of work, so please refer to the [blog post](ADD) for overlook of the main changes - like Flipper support and the new LogBox.
+This major release includes Flipper support by default, improved dark mode support, moving AppleTV to react-native-tvos, and more. See the [blog](https://reactnative.dev/blog/2020/03/24/version-0.62) for all of the highlights.
 
-As always, we’ve done our best to test this release at length, especially with this being such a large release. However, note that due to challenges caused by COVID-19 to people everywhere, we [the maintainers] may not have the bandwidth we’d like to have for quick follow-up on bugs and/or a patch release. When you run into an issue, please do try to provide a fix or a detailed ticket, but be prepared to revert to the version you were previously on to unblock yourself.
+We’re releasing this version to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
 
-For upgrading users, some of the progress comes with breaking changes; manual intervention may be required for your app - so remember to rely on the [upgrade-helper](https://react-native-community.github.io/upgrade-helper/) for a detailed breakdown of the needed changed.
-
-We are, in particular, aware that you may hit some issues on the Android side - for both, at the issue linked here, there's a workaround in place already:
-
-- [Release version instacrash on Android](https://github.com/facebook/react-native/issues/28163)
-- [APK size increases after Gradle version bump](https://github.com/facebook/react-native/issues/28330)
```

Comment:
My thinking for removing this is that this is better provided in the issue we link to for upgrade issues. That way we can edit/expand the links or close the issues without the changelog either needing changed or being stale.

This also teaches people to go to those issues for help instead of going to the changelog.

What do you think  @kelset?

Reply:
So the reason why we have been doing this kind of section for the last few 'minor' releases is that we have seen that out of the people upgrading, a significant number only check the changelog (or even only the release tab) before attempting an upgrade.

Because of that, in order to avoid 10s of issues reporting the same bug (because usually those people *also* don't check already opened issues) we have found that funnelling them straight from the changelog helps them towards the 'teaching' of being aware of the issue section.

In particular, for this version we have posted the links to those 2 because they are the two "flashiest" issues that have been reported and that are likely to create an emotional response (= going straight to the issue section and rage about).

So I'd prefer to keep this section in.

Reply:
> This also teaches people to go to those issues for help instead of going to the changelog.

I think release notes are a good place to note known issues. (While a ‘change log’ isn’t strictly the same as release notes, the pragmatist in me prefers supporting people being successful in self-servicing their issue rather than being strict with semantics.) I get what you’re saying regarding the note becoming stale, but I think that’s less of a problem than the overhead of people **not** succeeding in finding these known workarounds and _all_ parties losing time in the end.

So I’m in favour of highlighting known issues that are bound to be reported on otherwise, but think the overall text can probably be reduced and elaborated on in the linked release issue.

Reply:
Makes sense, updated!